pkg/pillar/types/domainmgrtypes.go

```diff
@@ -242,7 +242,7 @@ type VmConfig struct {
 	ExtraArgs  string // added to bootargs
 	BootLoader string // default ""
 	// For CPU pinning
-	CPUs string // default "", list of "1,2"
+	CPUs []int // default nil, list of [1,2]
```

Comment:
@roja-zededa , this change implies on change the API as well: https://github.com/lf-edge/eve-api/blob/main/proto/config/vm.proto#L48

I'm not sure how this impacts the controller.... I wouldn't change the type, you can still keep as string and parse it accordingly....

Reply:
+1 changes on API would require changes on controller

Reply:
Nikolay authored this commit.  @OhmSpectator  Could you please take a look at it?

Reply:
cmd/zedagent/parseconfig.go parses the received protobuf and produces the golang structs such as this. Thus it can be made to parse the existing protbuf string format and produce an array of integers.
Thus no need to change the EVE API.

Reply:
This field in the protobuf that @rene pointed to is not filled out by the controller at the moment. So we don't even have to change the parsing logic. It's currently used only internally in the DomainStatus.